lib/archiving.js

```diff
@@ -49,6 +49,17 @@ var request = require('request'),
 *          <a href="https://dashboard.tokbox.com">OpenTok dashboard</a>.
 *   </ul>
 *
+* @param {Boolean} hasAudio
+*   The archive has an audio track. If hasAudio is set to true when calling OpenTok.startArchive,
+*   an audio track will be inserted to the archive. hasAudio is an optional parameter that is set
+*   to true by default.
+*
+* @param {Boolean} hasVideo
+*   The archive has a video track.  If hasVideo is set to true when calling OpenTok.startArchive,
+*   a video track will be inserted to the archive. hasAudio and hasVideo cannot be both false
+*   when calling OpenTok.startArchive. hasVideo is an optional parameter that is set to true by
+*   default.
```

Comment:
These details should be mentioned in the documentation for OpenTok.startArchive(), not here. (Keep the first sentence only.)